docs/modules/ROOT/pages/reference/service-maturity.adoc

```diff
@@ -59,18 +59,19 @@ See also:
 As a user of the service I can:
 
 * see metrics of a service instance
+* see SLO reports
 * access a dashboard showing pre-crafted diagrams
```

Comment:
@Kidswiss could you please explain what exactly is meant by this? Do the 2 upper points possibly cover this?

Reply:
with this I mean "access a dashboard showing pre-crafted diagrams"

Reply:
That the customer has a link to a page where they can see visualizations of the available metrics.

Reply:
Would that not be the same as "see metrics of a service instance"?

Reply:
Not necessarily. I interpreted that one that we expose the raw metrics for the customer and they can scrape and visualize themselves. The dashboard with our pre-crafted diagrams would be added bonus so they don't have to do it themselves.

Reply:
That does sound good. Should we add "of the available metrics" at the end of "access a dashboard showing pre-crafted diagrams"? That way it would be clear what the graphs show.

Reply:
Yeah sounds good.

I would probably also re-write the other one to "see the raw metrics of a service instance". So it's clear to them that the raw metrics are also available if they wish.